mypy/fastparse.py

```diff
@@ -1111,15 +1114,17 @@ def visit_JoinedStr(self, n: ast3.JoinedStr) -> Expression:
     def visit_FormattedValue(self, n: ast3.FormattedValue) -> Expression:
         # A FormattedValue is a component of a JoinedStr, or it can exist
         # on its own. We translate them to individual '{}'.format(value)
-        # calls -- we don't bother with the conversion/format_spec fields.
-        exp = self.visit(n.value)
-        exp.set_line(n.lineno, n.col_offset)
-        format_string = StrExpr('{}')
+        # calls, passing along specified format specifiers and conversions.
```

Comment:
Add a quick comment indicating that mypyc is the motivation for caring about this